doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md

```diff
@@ -54,7 +54,7 @@ wget https://sourceforge.net/projects/boost/files/boost/1.55.0/boost_1_55_0.tar.
 tar -xf boost_1_55_0.tar.gz
 cd boost_1_55_0
 ./bootstrap.sh
-sudo ./bjam install
+sudo ./bjam cxxflags=-std=c++11 install
```

Comment:
The option is needed to enable 11th standard support for Ubuntu 16.04 default g++ compiler. It is not enabled by default and _boost_ does not pass the necessary flag implicitly.